system/xmllibtest/xmllibtest.cpp

```diff
@@ -317,7 +317,7 @@ void doTransform()
             {
                 if (gStreamingBySocket)
                 {
-                    SocketEndpoint ep(gPort,gHost);
+                    SocketEndpoint ep(gHost, gPort);
```

Comment:
NB: I think this this was accidental param order, however, it happened to be 'ok' because gHost would convert to an IpAddress and call the SocketEndpoint ctor that took port 1st..